libvast/builtins/loaders/stdin_loader.cpp

```diff
@@ -0,0 +1,102 @@
+//    _   _____   __________
```

Comment:
Unrelated to this line: Can you rename this file to be stdin.cpp so it's in line with the other builtins where the filename exactly matches the plugin name?

Reply:
I renamed the file to `stdin.cpp`. https://github.com/tenzir/vast/pull/2935/commits/709a347f8a2c22abba3180e8896f5769de03fbc5